### PR TITLE
Add new Rules documentation that covers YAML Editor view

### PR DESCRIPTION
**Pull request by cwillum**
Signed-off-by: cwillum <cwmmoore@amazon.com>

### Description
A new YAML Editor view has been added for 2.5 on the Rules page for Security Analytics.

### Issues Resolved
Added new content and added or updated screenshots to include the new feature.

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _security-analytics/usage/rules.md [hunk 1]
```diff
@@ -17,7 +17,27 @@ When you open the Rules page, all rules are listed in the table. Use the search
 Alternatively, you can use the **Rule type**, **Rule severity**, and **Source** dropdown menus to drill down in the list of alerts and filter for preferred results. You can use all three menus in combination to narrow results. Select only one option per menu.
```
Comment (AWSHurneyt) [inline _security-analytics/usage/rules.md:17]: > Select only one option per menu.

I must have missed this during my initial read of the documentation. The filter menus for this table support selecting multiple options (e.g., the table can be filtered to display only `Critical`, and `Low` severity rules).

File: _security-analytics/usage/rules.md [hunk 2]
```diff
@@ -17,7 +17,27 @@ When you open the Rules page, all rules are listed in the table. Use the search
...
+
+## Creating rules
+
+There are a number of ways to create rules on the Rules page. The first is to manually fill in the necessary fields that complete the rule, using either the Visual Editor or YAML Editor. To do this, select the **Create new rule** button in the uppper-right corner of the Rules window. The Create a rule window opens.
+<br><img src="{{site.url}}{{site.baseurl}}/images/Security/create-a-rule.png" alt="The Create a rule window, which includes the Visual Editor and YAML editor." width="500">
+
+If you choose to create the rule manually, you can refer to Sigma's [Rule Creation Guide](https://github.com/SigmaHQ/sigma/wiki/Rule-Creation-Guide) to help understand details for each field.
+* By default, the Visual Editor is displayed. Enter the appropriate content in each field and select **Create** in the lower-right corner of the window to save the rule.
+* the Create a rule window also provides the YAML Editor so that you can create the rule directly in a YAML file format. Select **YAML Editor** and then enter information for the pre-populated field types.
```
Comment (AWSHurneyt) [inline _security-analytics/usage/rules.md:37]: Small typo: Should this bullet point start capital `T`?

**Comment by cwillum** [comment]
Waiting a while longer for Doc Review to see if I can get a Tech Review first today.

File: _security-analytics/usage/rules.md (hunk 1 above)
Comment (cwillum) [reply _security-analytics/usage/rules.md:17]: Operator error. Yes, pick as many as you'd like. Thanks. Reworded this statement.

File: _security-analytics/usage/rules.md (hunk 2 above)
Comment (cwillum) [reply _security-analytics/usage/rules.md]: Yup. Thank you.

**Review by carolxob — APPROVED** [review]
LGTM.

**Review by AWSHurneyt — APPROVED** [review]
Looks good to me. I forwarded this PR to the rest of the security analytics team in case they have any feedback.

File: _security-analytics/usage/rules.md [hunk 3]
```diff
@@ -14,10 +14,30 @@ The Rules window lists all security rules and provides options for filtering the
...
+<br><img src="{{site.url}}{{site.baseurl}}/images/Security/Rule_details.png" alt="The rule details pane" width="500">
+
+In Visual view, rule details are arranged in fields, and the links are active. Select **YAML** to display the rule in YAML file format.
+<br><img src="{{site.url}}{{site.baseurl}}/images/Security/rule_detail_yaml.png" alt="The rule details pane in YAML file view" width="500">
+* Rule details are formatted as a YAML file according to the Sigma rule specification.
+* To copy the rule, select the copy icon in the top right corner of the rule. To quickly create a new and customized rule, you can paste the rule into the YAML editor and make any modifications before saving it. See [Customizing rules](#customizing-rules) for details.
+
+## Creating rules
+
+There are a number of ways to create rules on the Rules page. The first is to manually fill in the necessary fields that complete the rule, using either the Visual Editor or YAML Editor. To do this, select the **Create new rule** button in the uppper-right corner of the Rules window. The Create a rule window opens.
```
Comment (vagimeli) [inline _security-analytics/usage/rules.md:32]: To tighten wording, I suggest using "several" in place of "a number of"

Bold "Create a rule"?

File: _security-analytics/usage/rules.md [hunk 4]
```diff
@@ -14,10 +14,30 @@ The Rules window lists all security rules and provides options for filtering the
...
+<br><img src="{{site.url}}{{site.baseurl}}/images/Security/create-a-rule.png" alt="The Create a rule window, which includes the Visual Editor and YAML editor." width="500">
+
+If you choose to create the rule manually, you can refer to Sigma's [Rule Creation Guide](https://github.com/SigmaHQ/sigma/wiki/Rule-Creation-Guide) to help understand details for each field.
+* By default, the Visual Editor is displayed. Enter the appropriate content in each field and select **Create** in the lower-right corner of the window to save the rule.
+* The Create a rule window also provides the YAML Editor so that you can create the rule directly in a YAML file format. Select **YAML Editor** and then enter information for the pre-populated field types.
```
Comment (vagimeli) [inline _security-analytics/usage/rules.md:37]: Bold "Create a rule"?

File: _security-analytics/usage/rules.md [hunk 5]
```diff
@@ -14,10 +14,30 @@ The Rules window lists all security rules and provides options for filtering the
...
+
+The alternatives to manually creating a rule, however, make the process easier and faster. They involve either importing a rule in a YAML file or duplicating an existing rule and customizing it. See the next two sections for detailed steps.
```
Comment (vagimeli) [inline _security-analytics/usage/rules.md:39]: Per the style guide, we should avoid terms like "easier" Suggestion: "..., however, simplify and speed up the process."

File: _security-analytics/usage/rules.md [hunk 6]
```diff
@@ -57,7 +77,7 @@ status: experimental
 ```
 
 1. To begin, select the **Import rule** button in the upper-right corner of the page. The Import rule page opens.
-1. Either drag a YAML-formatted Sigma rule into the window or browse for the file by selecting the link and opening it. The Import a rule window opens and the rule definition fields are automatically populated.
+1. Either drag a YAML-formatted Sigma rule into the window or browse for the file by selecting the link and opening it. The Import a rule window opens and the rule definition fields are automatically populated in both the Visual Editor and YAML Editor.
```
Comment (vagimeli) [inline _security-analytics/usage/rules.md:80]: Bold "Import a rule"?

File: _security-analytics/usage/rules.md [hunk 7]
```diff
@@ -57,7 +77,7 @@ status: experimental
...
 1. Verify or modify the information in the fields.
 1. After you confirm the information for the rule is accurate, select the **Create** button in the lower-right corner of the window. A new rule is created, and it appears in the list of rules on the main page of the Rules window.
```
Comment (vagimeli) [inline _security-analytics/usage/rules.md:82]: Bold "Rules"?

File: _security-analytics/usage/rules.md [hunk 8]
```diff
@@ -67,10 +87,10 @@ An alternative to importing a rule is duplicating a Sigma rule and then modifyin
 <img src="{{site.url}}{{site.baseurl}}/images/Security/rules-dup1.png" alt="Selecting a rule in the Rules name list">
 
 1. To begin, select the rule in the Rule name column. The rule details pane opens.
```
Comment (vagimeli) [inline _security-analytics/usage/rules.md:89]: Bold "Rule name"?

File: _security-analytics/usage/rules.md [hunk 9]
```diff
@@ -67,10 +87,10 @@ An alternative to importing a rule is duplicating a Sigma rule and then modifyin
...
-<img src="{{site.url}}{{site.baseurl}}/images/Security/rule-dup3.png" alt="Selecting the duplicate button" width="400">
-1. Modify any of the fields to customize the rule.
+<br><img src="{{site.url}}{{site.baseurl}}/images/Security/rule-dup2.png" alt="Opening the rule details pane" width="500">
+1. Select the **Duplicate** button in the upper-right corner of the pane. The Duplicate rule window opens in Visual Editor view and all of the fields are automatically populated with the rule's details. Details are also populated in YAML Editor view.
+<br><img src="{{site.url}}{{site.baseurl}}/images/Security/dupe-rule.png" alt="Selecting the duplicate button opens the Duplicate rule window" width="500">
+1. In either Visual Editor view or YAML Editor view, modify any of the fields to customize the rule.
 1. After performing any modifications to the rule, select the **Create** button in the lower-right corner of the window. A  new and customized rule is created, and it appears in the list of rules on the main page of the Rules window.
```
Comment (vagimeli) [inline _security-analytics/usage/rules.md:94]: Replace double space between "A" and "new" with single space.

**Review by vagimeli — APPROVED** [review]
Minimal comments; LGTM

**Pull request merged**